packages/vue/src/aspect-ratio/AspectRatio.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+import type { PrimitiveProps } from '@oku-ui/primitive'
+
+export interface AspectRatioProps extends PrimitiveProps {
+  ratio?: number
+}
+</script>
+
+<script setup lang="ts">
+import { defineOptions } from 'vue'
+import { useComponentRef } from '@oku-ui/use-composable'
+import { Primitive } from '@oku-ui/primitive'
+
+defineOptions({
+  name: 'OkuAspectRatio',
```

Comment:
```suggestion
  name: 'OkuAspectRatio',
  inheritAttrs: false,
```

Reply:
since the first element is different and assigns $attrs to its child, we now have to make it false.